pkg/cmd/dev/cmd.go

```diff
@@ -29,74 +26,58 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const (
-	shortDesc = `Start development server (experimental)`
-	longDesc  = `Start development server (experimental) with the specified context directory and kitfile`
-	example   = `kit dev ./my-model --port 8080`
-)
-
-type DevOptions struct {
-	host       string
-	port       int
-	modelFile  string
-	contextDir string
-	configHome string
-	stop       bool
-}
-
-func (opts *DevOptions) complete(ctx context.Context, args []string) error {
-
-	configHome, ok := ctx.Value(constants.ConfigKey{}).(string)
-	if !ok {
-		return fmt.Errorf("default config path not set on command context")
-	}
-	opts.configHome = configHome
-	if !opts.stop {
-		opts.contextDir = ""
-		if len(args) == 1 {
-			opts.contextDir = args[0]
-		}
-		if opts.modelFile == "" {
-			foundKitfile, err := filesystem.FindKitfileInPath(opts.contextDir)
-			if err != nil {
-				return err
-			}
-			opts.modelFile = foundKitfile
-		}
-		if opts.host == "" {
-			opts.host = "127.0.0.1"
-		}
-
-		if opts.port == 0 {
-			availPort, err := findAvailablePort()
-			if err != nil {
-				output.Fatalf("Invalid arguments: %s", err)
-				return err
-			}
-			opts.port = availPort
-		}
+func DevCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "dev <directory> [flags]",
+		Short:   devShortDesc,
+		Long:    devLongDesc,
+		Example: devExample,
```

Comment:
Is example needed?

Reply:
I included it as an additional hint, since running `kit dev` just prints the help page now (whereas before it would start up the server):
```
❯ kit dev
Run models locally (experimental)

Description:
  Start a local server and interact with a model in the browser

Usage:
  kit dev [command]

Examples:
  kit dev start

Available Commands:
  logs        View logs for development server
  start       Start development server (experimental)
  stop        Stop development server
```

Reply:
:shipit: